src/Middleware/RateLimiting/src/Features/IRateLimiterContextFeature.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Connections.Features;
+
+namespace Microsoft.AspNetCore.RateLimiting.Features;
+
+public interface IRateLimiterContextFeature
```

Comment:
Note: there is no docs, cause i'm not sure about the API

Reply:
New APIs need to be approved. This needs an [API proposal](https://github.com/dotnet/aspnetcore/issues/new?assignees=&labels=api-suggestion&template=30_api_proposal.md&title=).

Reply:
The new APIs also need to be added to https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/RateLimiting/src/PublicAPI.Unshipped.txt. There's a suggestion hint in VS to do this for you.